UI/CSS/modal.css

```
@@ -0,0 +1,202 @@
+/*Modal*/
+.modal{
+	display: none;
+	position: fixed;
+	padding: 30px;
+	z-index: 1;
+	left: 0;
+	right: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: rgb(0,0,0,0.5);
+
+}
+
+#modal-content{
+<<<<<<< HEAD
+	width: 50%;
+	height: 80%;
+=======
+	width: 70%;
+	height: 78%;
+>>>>>>> b82a9865333eb05673522dcfede22cda702b06b3
```

Fix the merge conflicts
```suggestion
>>>>>>> b82a9865333eb05673522dcfede22cda702b06b3
```